src/docs/references/marketplace-assets/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Marketplace assets
+slug: marketplace-assets
+updated: 2022-05-09
+category: references
+ingress:
+  Reference documentation providing information on Marketplace assets.
+published: true
+---
+
+_Marketplace assets_ are a mechanism defining configuration data and
+content for a marketplace. They are most useful for storing client
+application configuration and content that is managed by a marketplace
+operator and changes relatively infrequently. Each _marketplace asset_
+is an object that defines a _path_ and _content_. The path gives the
+asset a name and allows organizing assets in a way similar to a file
+system, where assets can be organized in directory-like structure. The
+content of the asset is its data in one of the supported data types.
+
+At present, Flex supports only JSON data as assets. Support for images
+(JPEG, PNG, etc) is coming in the future.
+
+The marketplace assets are typically edited by a marketplace operator
+though Flex Console.
+
+**IMPORTANT** All marketplace assets are considered public and must not
+be used to store secret or otherwise sensitive information.
+
+> **Note:** Marketplace assets are in early development. At present,
+> Console allows only limited ability to manage the content of a single
+> marketplace asset - the JSON document describing the
+> [translation strings](/concepts/translations/) for a client
+> application. Stay tuned for more later on!
+
+For example, a marketplace may have assets with the following paths:
+
+```
+config/config.json
+config/translations.json
+```
```

Comment:
How about code block syntax "shell" and "tree" structure?

```shell
└── config
    ├── config.json
    └── translation.json
```

Ref: https://github.com/sharetribe/flex-docs/blob/17e319c330294749eeec0c240360f5e3eac6fee2/src/docs/tutorial-transaction-process/use-protected-data-in-emails/index.md?plain=1#L100

How it looks like in practice:
https://www.sharetribe.com/docs/tutorial/use-protected-data-in-emails/#update-translations